pylint/checkers/misc.py

```diff
@@ -23,20 +23,32 @@
 
 from pylint.interfaces import IRawChecker
 from pylint.checkers import BaseChecker
-from pylint.utils import OPTION_RGX
+from pylint.utils import OPTION_RGX, MessagesHandlerMixIn
 
 
-MSGS = {
-    'W0511': ('%s',
-              'fixme',
-              'Used when a warning note as FIXME or XXX is detected.'),
-    'W0512': ('Cannot decode using encoding "%s",'
-              ' unexpected byte at position %d',
-              'invalid-encoded-data',
-              'Used when a source line cannot be decoded using the specified '
-              'source file encoding.',
-              {'maxversion': (3, 0)}),
-}
+class ByIdDisabledMessagesChecker(BaseChecker):
+
+    """checks for messages that are disabled by id instead of symbol."""
+
+    __implements__ = IRawChecker
+
+    # configuration section name
+    name = 'miscellaneous'
+    msgs = {'I0023': ('%s',
+                      'message-disabled-by-id',
```

Comment:
Now that I think of it, we should also warn when ids are used for enabling messages, so this should also catch `# pylint: enable=C0103` for instance. In this case, it would also make sense to rename the check to something as in `use-symbolic-message-instead` or something along these lines. What do you think @hippo91 ? I think we should also have access to what ids were enabled as well.

Reply:
@PCManticore you are right. I'll do this ASAP.